cli/src/main/java/org/infinispan/cli/commands/troubleshoot/log/AccessLogParse.java

```diff
@@ -0,0 +1,115 @@
+package org.infinispan.cli.commands.troubleshoot.log;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandException;
+import org.aesh.command.CommandResult;
+import org.aesh.command.impl.completer.FileOptionCompleter;
+import org.aesh.command.option.Arguments;
+import org.aesh.command.option.Option;
+import org.aesh.command.option.OptionList;
+import org.infinispan.cli.commands.CliCommand;
+import org.infinispan.cli.converters.LocalDateTimeConverter;
+import org.infinispan.cli.impl.ContextAwareCommandInvocation;
+
+/**
+ * Parses multiple access log files and generate statistics.
+ *
+ * @author José Bolina
+ * @since 15.0
+ */
+@CommandDefinition(name = "log", description = "Parses access log file and summarizes statistics")
+public class AccessLogParse extends CliCommand {
+
+   @Option(shortName = 'o', description = "Operation to filter")
+   String operation;
+
+   @OptionList(shortName = 'x', description = "Operations to exclude")
+   List<String> excludeOperations;
+
+   @Option(shortName = 't', description = "List the N longest operations")
```

Comment:
Should we have a way to print all commands that were longer than a given duration as well?

Reply:
Added the option `-d` to provide a duration in ms.